tests/fixture_setup/_unittests.py

```diff
@@ -532,14 +532,14 @@ def __str__(self):
         else:
             return "{}={}".format(self.name, self.version)
 
-    @property
-    def version(self):
-        return self._version
-
     @property
     def is_auto_removable(self):
         return self.marked_install and self.is_auto_installed
 
+    @property
```

Comment:
is there a reason for this change?

Reply:
> is there a reason for this change?

Yeah, mypy was erroring out on the following setter.  It appeared to need the @property next to the setter.

Reply:
gotcha